project-proposals/stage-1/Trainpositioning.md

```diff
@@ -0,0 +1,125 @@
+
+## Who are the maintainers of the project (these will be the primary contacts for the OpenRail Association)?
+
+Infrabel:
+
+- glenn.vancalster@infrabel.be
+- kevin.goos@infrabel.be
+- ilya.quateau@infrabel.be
+- mathias.vandenauweele@infrabel.be
+
+## Which organizations are sponsoring/contributing to the project?
+
+Infrabel
+
+## Where is the code hosted?
+
+At the moment, on a gitlab server at Infrabel. We want to opensource it immediately under the openrail organisation.
+
+## Which exact repositories do you intend to transfer to the GitHub organization of the OpenRail Association?
+
+Originally, the repository contains a JSON API. We will split of the logic from the API and put it in a separate repository. The API wrapper is not very useful to opensource, so we will only opensource the library repository.
+
+
+## What is the project's main license?
+
+Apache License 2.0
+
+## What other licenses does the project use, e.g., for included 3rd-party code?
+
+
+We are still in the progress of simplifying the library, but at the moment, these packages are used with their respective licenses:
+
+- NetTopologySuite: BSD-3-Clause License
+- NodaTime: Apache License 2.0
+- Npgsql.NetTopologySuite: PostgreSQL License
+- Serilog: Apache License 2.0
+- Serilog.Sinks.Console: Apache License 2.0
+- DotSpatial.Projections: MIT License
+
+We don't immediately see any major issues.
+
+## Are any trademarks associated with the project?
+
+No
+
+## Does the project have a web site? Where is it? Are you ok with moving it to be hosted by the OpenRail Association?
+
+No, no website. We will foresee documentation and a separate .net project to run tests, both will also be open sourced.
+
+
+## What are the communication channels the project uses (such as mailing lists, Slack, IRC, etc.)?
+
+Git issues
+
+## What is the project's leadership team and decision-making process?
+
+- PM: glenn.vancalster@infrabel.be
+- Dev: kevin.goos@infrabel.be
+- Dev: ilya.quateau@infrabel.be
+- Prev PM / Dev: mathias.vandenauweele@infrabel.be
+
+## How is it decided if and when a pull request is merged?
+
+Depends on the pull request, generally the project leader decides.
+
+## How can someone become a committer or a maintainer to/of the project?
+
+After discussion with the project leader and current development team.
+
+## How is development of the project planned and organized? Is this transparent to the public?
+
+Using git issues & epics/stories.
+
+## What is the project's roadmap?
+
+After the project refactoring and extensive testing, no further development is planned.
+
+## What other organizations in the world should be interested in this project?
+
+All organisations that use measurement trains and process the measurements. But the project can also be used as a guideline for organisations that want to post processes location measurements for other domains that include topology and navigability.

```

Comment:
Measurements done by a measurement train need to be linked to a track position. The TP-lib helps with this aspect of positioning. 

A measurement train will generally measure a value and save that value either in a kind of ledger together with other measurements such as gnss position. Or it could save the value together with the timestamp in a seperate file. _In a post processing step, the gnss position is improved and positioned on the track topology_. Afterwards, the measured value is positioned on the track topology.
The TP-lib helps with the italic step.